LEAF_Nexus/js/employeeSelector.js

```diff
@@ -157,24 +157,24 @@ employeeSelector.prototype.search = function() {
 		            	t.selection = '';
 		            	$('#' + t.prefixID + 'result').html('');
 		            	var buffer = '';
-		            	if(t.outputStyle == 'micro') {
+		            	if(t.outputStyle == 'standard') {
```

Comment:
Is there a reason this was swapped? This will cause all the micro's to become standard and the standard's to become micro... Why are we making a huge frontend swap like this?

Reply:
The reason I changed it was because that was the name that was given in the output style, I can change it if it wasn't meant to be changed.